reporting/reporting_calm_transformer/src/transform.py

```diff
@@ -0,0 +1,45 @@
+import math
+from copy import deepcopy
+from dateutil.parser import parse
+
+
+def convert_date_to_iso(date_string):
+    try:
+        return parse(date_string).date().isoformat()
```

Comment:
You should check if the Calm date ever uses "day-first" like in the UK in a way that could be ambiguous.

See https://notebook.alexwlchan.net/2018/10/beware-ambiguous-dates-with-dateutil-parse/